presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -225,11 +225,45 @@ public void testWithDifferentFilterCondition()
     }
 
     @Test
-    public void testWithNoGroupByInBaseQuery()
+    public void testWithGroupBy()
```

Comment:
If you mean queries like ```SELECT SUM(a), b AS b_alias FROM base GROUP BY b_alias```, they will not be optimized because `b` is not in `GROUP BY`.
Actually, we cannot use alias in `GROUP BY`, it is an illegal query so it doesn't matter.